pymfe/_internal.py

```diff
@@ -513,14 +518,14 @@ def _extract_mtd_args(
 
 
 def summarize(
-    features: t.Union[np.ndarray, t.Sequence],
+    features: t.Union[np.ndarray, t.List],
     callable_sum: t.Callable,
     callable_args: t.Optional[t.Dict[str, t.Any]] = None,
-) -> t.Union[t.Sequence, TypeNumeric]:
+) -> t.Union[t.List, TypeNumeric]:
     """Returns ``feature`` values summarized by ``callable_sum``.
 
     Args:
```

Comment:
I only realized it now, but we should use "Parameters" instead of "Args" to avoid creating automatic documentation problems.

We would never realize this because this file is not sent to automatic documentation.

Reply:
You don't have to fix this now. I can fix it later.